susemanager-utils/supportutils-plugin-susemanager-client/susemanagerclient

```diff
@@ -54,8 +53,7 @@ conf_files $OF \
     /etc/salt/minion \
     /etc/salt/minion.d/susemanager.conf \
     /etc/salt/minion.d/_schedule.conf \
-    /etc/uyuni/proxy/config.yaml \
-    /etc/sysconfig/uyuni-proxy-systemd-services
+    /etc/uyuni/proxy/config.yaml
```

Comment:
besides the testsuit i don't see any references to this config file. Is this used somewhere?

Reply:
no idea. I only dropped `/etc/sysconfig/uyuni-proxy-systemd-services` and removed the `\` at the end of the previous line